src/overlays/actors/ovl_Obj_Kinoko/z_obj_kinoko.h

```diff
@@ -11,4 +11,6 @@ typedef struct ObjKinoko {
 
 extern const ActorInit Obj_Kinoko_InitVars;
 
+#define OBJ_KINOKO_PARAMS(thisx) (thisx->params & 0x7F) 
```

Comment:
Since the sole use of this is to set the flag, probably call it `OBJ_KINOKO_GET_FLAG`?